Actual Budget Update Failing

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

actualbudget.sh

### 📂 What was the exact command used to execute the script?

update

### 📝 Provide a clear and concise description of the issue.

The Actual Budget 'update' command is failing.  This was occurring on my LXC that I've been running for quite a while from the ttech days (RIP ttech), so I spun up a new LXC last week and the issue is still happening in the new LXC.

Current Actual Budget version is 25.1.0

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 🔄 Steps to reproduce the issue.

open LXC console
run 'update' command

### ❌ Paste the full error output (if available).

.:mv: cannot stat 'actual-server-*': No such file or directory

[ERROR] in line 54: exit code 0: while executing command mv acutal-server-* /opt/actualbudget

### 🖼️ Additional context (optional).

Commit history leads me to believe that this issue may have been recently introduced as the corresponding lines have recent changes.

@MickLesk followed your instructions both in existing LXC and new LXC.  While the mv issue seems to be fixed in the new LXC, I'm now seeing the following error:

`[ERROR] in line 53: exit code 0: while executing command yarn install &> /dev/null`

which seems to be what @infinitytec was seeing in [#1623](https://github.com/community-scripts/ProxmoxVE/issues/1623)

Yep.  When I then try to do the corepack update suggested on infinitytec's issue report, I now get a different move error:

```
mv cannot stat '/opt/actualbudget_bak/.env': no such file or directory
[ERROR] in line 50: exit code 0: while executing command mv /opt/actualbudget_bak.env /opt/actualbudget
```
Might I request the issue be re-opened to increase visibility so that other devs/users can take a look?

I get the following error when executing `update` or executing the script:
`⠇mv: cannot move '/opt/actualbudget' to '/opt/actualbudget_bak/actualbudget': Directory not empty

[ERROR] in line 46: exit code 0: while executing command mv /opt/actualbudget /opt/actualbudget_bak`

> Yep. When I then try to do the corepack update to solve the yarn install issue, as suggested on infinitytec's issue report, I now get a different move error:
> 
> ```
> mv cannot stat '/opt/actualbudget_bak/.env': no such file or directory
> [ERROR] in line 50: exit code 0: while executing command mv /opt/actualbudget_bak.env /opt/actualbudget
> ```
> 
> Might I request the issue be re-opened to increase visibility so that other devs/users can take a look?

@MickLesk can we get this issue re-opened?

--- a/ct/actualbudget.sh
+++ b/ct/actualbudget.sh
@@ -44,9 +44,8 @@ function update_script() {
         cd /tmp
         wget -q https://github.com/actualbudget/actual-server/archive/refs/tags/v${RELEASE}.tar.gz
         mv /opt/actualbudget /opt/actualbudget_bak
-        mkdir -p /opt/actualbudget/
         tar -xzf v${RELEASE}.tar.gz >/dev/null 2>&1
-        mv *ctual-server-*/* /opt/actualbudget
+        mv *ctual-server-* /opt/actualbudget
         rm -rf /opt/actualbudget/.env
         mv /opt/actualbudget_bak/.env /opt/actualbudget
         mv /opt/actualbudget_bak/server-files /opt/actualbudget/server-files

--- a/ct/actualbudget.sh
+++ b/ct/actualbudget.sh
@@ -60,7 +60,7 @@ function update_script() {
         
         msg_info "Cleaning Up"
         rm -rf /opt/actualbudget_bak
-        rm -rf /tmp/actual-server.tar.gz
+        rm -rf /tmp/v${RELEASE}.tar.gz
         msg_ok "Cleaned"
         msg_ok "Updated Successfully"
     else

